scripts/simorghPages.js

```diff
@@ -64,7 +64,7 @@ const generateLinks = (service, env, domain) => {
   return output.join('<br/>');
 };
 
-const generateLaunchDates = (service) => {
+const generateLaunchDates = service => {
```

Comment:
Bit confused why this change is needed on this PR? Won't latest be failing the prettier check if we need to make this change?

Reply:
I've seen this before where my es-lint on Atom has picked this file up before. It wouldn't let me push without a `-no-verify` unless I fixed it. So I can remove it and create a separate issue for it. Not sure why this check isn't failing on latest 🤷‍♂️.

Reply:
No big deal @HarveyPeachey , just didn't make any sense was all. Feel free to leave it in the PR imo.

Reply:
In regards to the chromatic snapshots. Just checked the bottom padding snapshot and checked both build snapshot canvas's and the sizes are the same, so I'm assuming chromatic bugged out generating the image.

The spacing does look weird, but this existed before the PR. I agree all items should share the same height so it doesn't look skewed like that. Maybe it was a design choice to get the component out faster but not sure.

Reply:
> Just checked the bottom padding snapshot and checked both build snapshot canvas's and the sizes are the same
Cool thanks for checking I'll approve on that basis 👍 

Could you show a UXer the spacing anamoly and raise an issue if they think it's wrong?

Reply:
Yeah, good shout. I'll also ask the people who worked on this in the home pod.